cclib/method/nuclear.py

```diff
@@ -148,46 +148,46 @@ def moment_of_inertia_tensor(self, atomcoords_index=-1):
 
         return moi_tensor
 
-    def principal_moments_of_inertia(self, units='amu_bohr_2'):
+    def principal_moments_of_inertia(self, units="amu_bohr_2"):
         """Return the principal moments of inertia in 3 kinds of units:
         1. [amu][bohr]^2
         2. [amu][angstrom]^2
         3. [g][cm]^2
         and the principal axes.
         """
-        choices = ('amu_bohr_2', 'amu_angstrom_2', 'g_cm_2')
+        choices = ("amu_bohr_2", "amu_angstrom_2", "g_cm_2")
         units = units.lower()
         if units not in choices:
             raise ValueError("Invalid units, pick one of {}".format(choices))
         moi_tensor = self.moment_of_inertia_tensor()
         principal_moments, principal_axes = np.linalg.eigh(moi_tensor)
-        if units == 'amu_bohr_2':
-            conv = 1
-        if units == 'amu_angstrom_2':
+        if units == "amu_bohr_2":
             _check_scipy(_found_scipy)
-            bohr2ang = spc.value('atomic unit of length') / spc.angstrom
-            conv = bohr2ang ** 2
-        if units == 'g_cm_2':
+            bohr2ang = spc.value("atomic unit of length") / spc.angstrom
+            conv = 1 / bohr2ang ** 2
+        if units == "amu_angstrom_2":
+            conv = 1
```

Comment:
So I swapped the two cases and forgot to take the inverse of one of them...good job past me.